modules/bridge/src/main/ts/ephox/bridge/components/menu/FancyMenuItem.ts

```diff
@@ -23,7 +25,8 @@ const fancyTypes: (keyof FancyActionArgsMap)[] = [ 'inserttable', 'colorswatch'
 export const fancyMenuItemSchema = ValueSchema.objOf([
   FieldSchema.strictString('type'),
   FieldSchema.strictStringEnum('fancytype', fancyTypes),
-  FieldSchema.defaultedFunction('onAction', Fun.noop)
+  FieldSchema.defaultedFunction('onAction', Fun.noop),
+  FieldSchema.defaulted('initData', undefined)
```

Comment:
```suggestion
  FieldSchema.defaulted('initData', {})
```
To match my comment above.